Rover/Parameters.cpp

```diff
@@ -344,6 +344,13 @@ const AP_Param::Info Rover::var_info[] = {
     GOBJECT(_gcs,           "MAV",  GCS),
 #endif
 
+    // @Param: FS_LEAK_ENABLE
+    // @DisplayName: Leak Failsafe Enable
+    // @Description: Controls what action to take if a leak is detected.
+    // @Values: 0:Warn only,1:RTL,2:Hold,3:SmartRTL,4:SmartRTL_Hold,5:Terminate,6:Loiter_Hold
```

Comment:
Thanks for this.  For the Values it's OK to have spaces.  So we can make "SmartRTL_Hold" into "SmartRTL or Hold".  Similarly "Loiter or Hold"